bokehjs/src/coffee/core/has_props.coffee

```diff
@@ -388,6 +388,7 @@ export class HasProps extends Backbone.Model
       @document._notify_change(@, attr, old, new_)
 
   materialize_dataspecs: (source) ->
+    # Note: this should be moved to a function separate from HasProps
```

Comment:
At some point our tooling will support mixin composition pattern, so this is a nice candidate to be moved to a trait.